src/main/java/com/example/rebrain/services/CardService.java

```diff
@@ -15,14 +18,17 @@
 @RequiredArgsConstructor
 public class CardService {
 
+    private final CardsSetsRepo cardsSetsRepo;
     private final CardRepo cardRepo;
 
-    public CardEntity create(CardEntity cardEntity) {
+    public CardEntity create(CardEntity cardEntity, Integer setId) {
         Integer userId = ThreadLocalUserIdHolder.get();
         cardEntity.setUserId(userId);
         log.debug("Saving new card: {}", cardEntity);
         CardEntity savedEntity = cardRepo.save(cardEntity);
         log.info("Card saved with ID: {}", savedEntity.getId());
+        CardsSetsEntity cardsSetsEntity = new CardsSetsEntity(cardEntity.getId(), setId);
+        cardsSetsRepo.save(cardsSetsEntity);
         return savedEntity;
     }
```

Comment:
We need to make it in single transaction otherwise we are gonna get data inconsistency issues